source/octf/interface/TraceManager.cpp

```diff
@@ -267,6 +268,16 @@ int64_t TraceManager::getTraceSize() const {
     return result;
 }
 
+int32_t TraceManager::getTraceVersion() const {
+    int32_t version = 0;
+
+    if (m_jobs.size()) {
+        version = m_jobs[0]->getTraceVersion();
+    }
```

Comment:
Exception when size equals 0

Reply:
Summary is also created when job count isn't initialized, so this exception can't exist